docs/HowToUpdateDocumentation.md

```diff
@@ -1,8 +1,18 @@
 ﻿
 # How to: Update Documentation
 
+The following documentation for project in can be updated:
```

Comment:
This sentence does not make sense to me.

Reply:
Me neither.